account_general_ledger_report/models/general_ledger_report.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+import datetime
+from openerp import models, api, fields, _
+
+
+class AccountGeneralLedgerReport(models.Model):
+    _name = 'account.general.ledger.report'
+
+    name = fields.Char(
+        string='Name',
+        readonly=True,
+    )
+    fiscalyear_id = fields.Many2one(
+        'account.fiscalyear',
+        string='Fiscal Year',
+        readonly=True,
+    )
+    target_move = fields.Selection(
+        [('posted', 'All Posted Entries'),
+         ('all', 'All Entries')],
+        string='Target Moves',
+        readonly=True,
+    )
+    reconcile_cond = fields.Selection(
+        [('all', 'All Items'),
+         ('open_item', 'Open Items'),
+         ('reconciled', 'Full Reconciled')],
+        string=" ",
```

Comment:
No string?

Reply:
ใช่ครับ ต้องการให้หน้า filter เป็นค่าว่าง
![selection_007](https://user-images.githubusercontent.com/20896369/43752822-eb7c8b9c-9a2c-11e8-979e-252b6d18d805.png)

Reply:
In <field .../> you can use nolabel="1". Or use <div ...>
Pleae find other place for example, there are plenty.
Note: using string=" " make code not beautiful and not reusable in other case.

Reply:
Sample here,

![selection_488](https://user-images.githubusercontent.com/1973598/43753790-9e4d9280-9a31-11e8-81e8-56077287a1f1.png)